Greenfin17/swimmer-buddy
# adding authorization and navbar

## Description
Add
-Navbar with login, logout buttons
-Authorization functionality

## Related Issue
https://github.com/Greenfin17/swimmer-buddy/issues/29

## Motivation and Context
App requires authorization as workouts and collections are tied to the creator.

## How Can This Be Tested?
On page load, a navbar with a login button should appear
When the user clicks the login button, they should see a google pop-up for login.
If the user is successful in signing in with Google, the user should see a logout button on the navbar.

## Types of changes
